panel/widgets/tables.py

```diff
@@ -1560,18 +1571,19 @@ def _update_selected(self, *events: param.parameterized.Event, indices=None):
         if self.pagination == 'remote' and self.value is not None:
```

Comment:
Note related to your work, just adding a comment to make my life easier!

```suggestion
        if self.pagination == 'remote' and self.value is not None:
            # Compute integer indexes of the selected rows
            # on the displayed page
```